src/main/java/org/alfresco/repo/domain/node/NodeEntity.java

```diff
@@ -334,18 +335,30 @@ public synchronized void setAuditableProperties(AuditablePropertiesEntity audita
     {
         checkLock();
         this.auditableProperties = auditableProperties;
-    }
-
-    @Override
-    public String getShardKey()
-    {
-        return this.shardKey;
-    }
-
-    public synchronized void setShardKey(String shardKey)
-    {
-        this.shardKey = shardKey;
-    }
-    
-    
+    }
+
+    @Override
+    public String getShardKey()
+    {
+        return this.shardKey;
+    }
+
+    @Override
+    public Integer getExplicitShardId()
+    {
+        return this.explicitShardId;
+    }
+
+    public synchronized void setShardKey(String shardKey)
+    {
+        this.shardKey = shardKey;
+    }
+
+    public synchronized void setExplicitShardId(Integer explicitShardId)
+    {
+         this.explicitShardId = explicitShardId;
+    }
+
+
+

```

Comment:
Only including "getExplicitShardId" and "setExplicitShardId" methods, but changes are affecting other parts of the code.